### PR TITLE
Terms for partners

### PR DESCRIPTION
**Pull request by MariekeGO**

### CODE REVIEW

File: dataminer-overview/Legal/Commercial_Terms/Terms_partners.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+uid: Terms_partners
+---
+
+# Terms for partners
+
+When submitting the partnership application on Skyline Communications' website, by selecting the checkbox, you confirm that you have read, have understood, and agree to be bound by the Partner Program Terms below, including any linked policies. This action constitutes a legally binding acceptance of the terms under Belgian law.
```
Comment (MariekeGO) [inline dataminer-overview/Legal/Commercial_Terms/Terms_partners.md:7]: @michelhofmans, I have rephrased the end of your document as the introduction here. Does this look OK for you?

File: dataminer-overview/Legal/Commercial_Terms/Terms_partners.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Skyline Communications nv\
+Ambachtenstraat 33\
+8870 Izegem\
+Belgium\
+Version 1.0, Tuesday, September 17, 2024
```
Comment (MariekeGO) [inline dataminer-overview/Legal/Commercial_Terms/Terms_partners.md:13]: Do we need to keep this here, or can it be removed?

File: dataminer-overview/Legal/Commercial_Terms/Terms_partners.md (hunk 1 above)
Comment (michelhofmans) [reply dataminer-overview/Legal/Commercial_Terms/Terms_partners.md:7]: Good, thank you.

File: dataminer-overview/Legal/Commercial_Terms/Terms_partners.md (hunk 2 above)
Comment (michelhofmans) [reply dataminer-overview/Legal/Commercial_Terms/Terms_partners.md:13]: Can be removed

**Review by michelhofmans — APPROVED** [review]

**Review by michelhofmans — APPROVED** [review]

**Pull request merged**